src/component/PageTabs/PageTabs.tsx

```diff
@@ -0,0 +1,38 @@
+import cn from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tabs: Tab[];
+};
+
+export const PageTabs: React.FC<Props> = ({ tabs }) => {
+  const { tabId } = useParams();
+  const selectedTab = tabs.find(tab => tab.id === tabId);
+
+  return (
+    <div className="section">
+      <div className="container">
+        <h1 className="title">Tabs page</h1>
+
+        <div className="tabs is-boxed">
+          <ul>
+            {tabs.map(tab => (
+              <li
+                key={tab.id}
+                data-cy="Tab"
+                className={cn({ 'is-active': tab.id === tabId })}
+              >
+                <Link to={`/tabs/${tab.id}`}>{tab.title}</Link>
```

Comment:
You can just use like this
```suggestion
                <Link to={`../${tab.id}`}>{tab.title}</Link>
```

Reply:
ok... thanks!